canonicalwebteam/discourse/app.py

```diff
@@ -308,14 +308,16 @@ def __init__(
         self.additional_metadata_validation = additional_metadata_validation
         pass
 
-    def get_index(self):
+    def get_index(self, limit=50, offset=0):
```

Comment:
The site works reasonably quickly with a limit of 500, but I wonder if we should enforce some maximum. Food for thought

Reply:
Do you mean keeping it at `limit=500` by default?

Reply:
Good point. I'll add an exception and a max limit of 500